src/components/Modal/Modal.styled.ts

```diff
@@ -92,36 +111,9 @@ export const SContent = styled.div<{
   z-index: ${theme.zIndices.modal};
 
   display: flex;
+  flex-flow: column;
   overflow: hidden;
 
-  background: ${theme.colors.darkBlue700};
```

Comment:
what's a reason of removing the gradient border?

Reply:
wasn't working well with currency reserves design, it looks according to design without this shadow tbh 


![Screenshot 2023-08-21 at 11 49 35](https://github.com/galacticcouncil/HydraDX-ui/assets/19366687/3e731ec7-feec-4407-a3df-e785e952862c)